package.json

```
@@ -1,11 +1,12 @@
 {
   "name": "@marfeel/adserver-providers-prebid-library",
-  "version": "3.23.0-pre-marfeel-2",
+  "version": "3.23.1-pre-marfeel-2",
```

We try to keep the original Prebid library version and append a `-marfeel-X` for our changes.

```suggestion
  "version": "3.23.0-pre-marfeel-3",
```

---

Fixed